data/tlg4102/tlg001/__cts__.xml

```diff
@@ -1,8 +1,8 @@
 <ti:work xmlns:ti="http://chs.harvard.edu/xmlns/cts" groupUrn="urn:cts:greekLit:tlg4102" xml:lang="grc" urn="urn:cts:greekLit:tlg4102.tlg001">
   <ti:title xml:lang="lat">Catena In Matthaeum (Catena Integra) (E Cod. Paris. Coislin. Gr. 23)</ti:title>
   <ti:edition urn="urn:cts:greekLit:tlg4102.tlg001.opp-grc1" workUrn="urn:cts:greekLit:tlg4102.tlg001">
-    <ti:label xml:lang="lat">Catena In Matthaeum (Catena Integra) (E Cod. Paris. Coislin. Gr.</ti:label>
-    <ti:description xml:lang="mul">Catenae (Novum Testamentum), Catena In Matthaeum (Catena Integra) (E Cod. Paris. Coislin. Gr. 23), 
-    Catenæ Græcorum Patrum in Novum Testamentum, Vol 1, Cramer, Oxford 1840</ti:description>
+    <ti:label xml:lang="lat">Catena In Matthaeum (Catena Integra) (E Cod. Paris. Coislin. Gr.)</ti:label>
+    <ti:description xml:lang="mul">Catenae (Novum Testamentum).  Catenæ Græcorum Patrum in Novum Testamentum, Vol 1.
```

Comment:
We don't generally use the dipthong (we would just transcribe to ae), but if you do, then I think you may need to have a Unicode declaration in the file.